4.5.c Release Notes (#5263)
spectrocloud/librarium: docs/docs-content/release-notes/release-notes.md

  Palette's network ports.

- Palette now uses Velero version 1.15 internally. This change allows newly deployed clusters to use [Kopia](https://velero.io/docs/main/file-system-backup/#how-velero-integrates-with-kopia) as the
  default backup and restore tool. Existing clusters will continue to use Restic as the default backup and restore tool.
  Refer to the [Backup and Restore](../clusters/cluster-management/backup-restore/backup-restore.md) page to learn more
  about backup and restore tools in Palette.

- The Self-hosted Palette Helm Chart installation method now supports custom image pull secrets. You can use the `global.imagePullSecrets` parameter in the Helm Chart **values.yaml** file to specify a [Docker configuration JSON object](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/#registry-secret-existing-credentials) containing your private registry credentials. Refer to the [Image Pull Secret](../enterprise-version/install-palette/install-on-kubernetes/palette-helm-ref.md#image-pull-secret) section of the Palette Helm Chart documentation to learn more about using custom image pull secrets.

### Edge

#### Features

> - Palette now supports managing [Amazon EKS Hybrid Nodes](https://aws.amazon.com/eks/hybrid-nodes/). Once your Amazon

Reviewer: ```suggestion
- Palette supports managing [Amazon EKS Hybrid Nodes](https://aws.amazon.com/eks/hybrid-nodes/). Once your Amazon
```

  self-hosted VerteX installations to migrate to the new architecture. Migrating to the new architecture removes the
  need for a dedicated file server to host the SCAR content. Check out the VerteX
  [Migrate SCAR to OCI Registry](../vertex/system-management/scar-migration.md) guide to learn more about migrating to
  the new SCAR architecture.

- The Self-hosted VerteX Helm Chart installation method now supports custom image pull secrets. You can use the
  `global.imagePullSecrets` parameter in the Helm Chart **values.yaml** file to specify a
  [Docker configuration JSON object](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/#registry-secret-existing-credentials)
  containing your private registry credentials. Refer to the
  [Image Pull Secret](../vertex/install-palette-vertex/install-on-kubernetes/vertex-helm-ref.md#image-pull-secret)
  section of the Palette Helm Chart documentation to learn more about using custom image pull secrets.

### Automation

#### Breaking Changes

- The Palette CLI now requires an encryption passphrase for various commands. The passphrase can be set as an

>   environment variable or passed as a flag to the CLI command. The passphrase encrypts and decrypts sensitive data, such

Reviewer: ```suggestion
  environment variable or using a CLI command flag. The passphrase encrypts and decrypts sensitive data, such
```

  as secrets, in the CLI configuration files. Refer to the
  [Palette CLI Encryption](../automation/palette-cli/palette-cli.md#encryption) section to learn more about the
  encryption passphrase.

#### Features

- Terraform version 0.22.3 of the
  [Spectro Cloud Terraform provider](https://registry.terraform.io/providers/spectrocloud/spectrocloud/latest/docs) is
  available. For more details, refer to the Terraform provider
  [release page](https://github.com/spectrocloud/terraform-provider-spectrocloud/releases).

> - A new command is now available in the Palette CLI. The command `validate-auth` can be used to validate you meet all

Reviewer: ```suggestion
- A new command has been added to the Palette CLI. The command `validate-auth` can be used to validate you meet all
```

>   permission required to deploy a target cluster into an infrastructure provider environment, such as AWS or Azure.

Reviewer: ```suggestion
  permissions required to deploy a target cluster into an infrastructure provider environment, such as AWS or Azure.
```
